soroban-sdk/src/env.rs

```diff
@@ -244,33 +249,13 @@ impl Env {
     pub fn log_value<V: IntoVal<Env, RawVal>>(&self, v: V) {
         internal::Env::log_value(self, v.into_val(self));
     }
-
-    #[doc(hidden)]
-    pub fn get_ledger_version(&self) -> u32 {
-        internal::Env::get_ledger_version(self).try_into().unwrap()
-    }
-
-    #[doc(hidden)]
-    pub fn get_ledger_sequence(&self) -> u32 {
-        internal::Env::get_ledger_sequence(self).try_into().unwrap()
-    }
-
-    #[doc(hidden)]
-    pub fn get_ledger_timestamp(&self) -> u64 {
-        let obj = internal::Env::get_ledger_timestamp(self);
-        internal::Env::obj_to_u64(self, obj)
-    }
-
-    #[doc(hidden)]
-    pub fn get_ledger_network_id(&self) -> Bytes {
-        let bin_obj = internal::Env::get_ledger_network_id(self);
-        bin_obj.in_env(self).try_into().unwrap()
-    }
 }
 
 #[cfg(feature = "testutils")]
 use crate::testutils::ContractFunctionSet;
 #[cfg(feature = "testutils")]
+use soroban_env_host::LedgerInfo;
```

Comment:
🤔 It doesn't look like LedgerInfo is exported, so I don't think contracts can actually call `set_ledger` with it.

Reply:
Exported in 66797858c05123e6f426b20a093403dc5dd77372.